HelloKelly/HelloKelly/Program.cs

```diff
@@ -8,15 +8,21 @@ namespace HelloKelly
 {
     class Program
     {
+        
         static void Main(string[] args)
         {
             string userName;
             Console.WriteLine("What is your name?");
             userName=Console.ReadLine();
-            Console.WriteLine("Nice to meet you, " + userName + "!");
+            Greeting(userName);
             Console.Write("Press <Enter> to exit... ");
             while (Console.ReadKey().Key != ConsoleKey.Enter) { }
```

Comment:
You can simplify this to Console.ReadLine(). It's the typical convention used to prevent the console from closing until you press Enter.